### ISSUE TITLE
Script can't access macros in AdditionalMacros.txt

### ISSUE THREAD

**Issue by Wermine** [PoE-TradeMacro/POE-TradeMacro#242]
I have AdditionalMacros.txt in trademacro's install folder and in c:\Users\<my name>\Documents\PoE-TradeMacro\ and both files have this line:

`F5::SendInput {Enter}/hideout{Enter}		; go to hideout with F5`

Nevertheless F5 doesn't work in game. Trademacro itself works. Pause key does enable and disable item parsing (which is also defined in AdditionalMacros.txt, perhaps defined in some other place too?).

If I right-click systray's icon and select Edit -> Additional Macros, I get Error - File not found. If I click Open User Folder, I get File Explorer's quick access folder.

**Comment by AliBabo** [comment]
Same here

**Comment by Eruyome** [comment]
Does any other hotkey in AdditionalMacros work? Have to check if the pause key is definned somewhere else, it once was but was moved to additional macros instead.

I may have to read up a bit on that Win 10 File Explorer quick access folder, can't test it myself since I'm on Win 8.

**Comment by Eruyome** [comment]
This should open the AdditionalMacros.txt in the user folder if it exists. Is the returned path right for you?

```
#SingleInstance force
#NoEnv ; Recommended for performance and compatibility with future AutoHotkey releases.
SetWorkingDir, %A_ScriptDir%

userDirectory := A_MyDocuments . "\PoE-TradeMacro"
Filename := "AdditionalMacros.txt"

Filepath := userDirectory . "\" . Filename
Msgbox % FilePath
IfExist, % Filepath
{
	Run, % Filepath
}
```

**Comment by Wermine** [comment]
It indeed opens correct AdditionalMacros.txt and correct folder. And I can still see the 

`F5::SendInput {Enter}/hideout{Enter} ; go to hideout with F5`

line on that opened file.

**Comment by Eruyome** [comment]
Try this here, I added some debug output. A console window should open and show this information, one block at scripstart, the other two when using the menu entries.
https://dl.dropboxusercontent.com/u/13620316/POE-TradeMacro.zip

# Scriptstart #
ScriptDir: D:\Users\Eru\Documents\GitHub\POE-TradeMacro
projectName: PoE-TradeMacro
userDirectory: D:\Users\Eru\Documents\PoE-TradeMacro

# Open AdditionalMacros #
userDirectory :D:\Users\Eru\Documents\PoE-TradeMacro
OpenFile: D:\Users\Eru\Documents\PoE-TradeMacro\AdditionalMacros.txt

# Open Settings Folder #
OpenFolder: D:\Users\Eru\Documents\PoE-TradeMacro

**Comment by Wermine** [comment]
I downloaded and ran that. Everything works. I then closed that AHK script, tried old Run_TradeMacro.ahk with admin privileges and that works too. I can access user folder via systray, I can access AdditionalMacros.txt via systray and those macros work in game. Font is monospaced, as it shoud.

I might've been starting wrong scripts without privileges? I'm sorry, but it's a bit messy with .exe file and several ahk files.
